tests/manage/test_pvc_disruptive.py

```diff
@@ -21,43 +61,171 @@ class BaseDisruption(ManageTest):
     pvc_obj = None
     storage_type = None
 
+    def create_pvcs(self):
+        """
+        Create PVCs
+
+        Returns:
+            list: List of PVC objects
+        """
+        # Parameters for PVC yaml as dict
+        pvc_data = load_yaml_to_dict(constants.CSI_PVC_YAML)
+        pvc_data['metadata']['namespace'] = self.namespace
+        pvc_data['spec']['storageClassName'] = self.sc_obj.name
+        pvc_data['metadata']['name'] = helpers.create_unique_resource_name('test', 'pvc')
```

Comment:
IIUC, this will create 5 PVCs with the same name

Reply:
Unique name will be given to each PVC in create_multiple_pvc.